e2e/test/scenarios/dashboard-cards/reproductions/22265-broken-added-series-prevents-editing-card.cy.spec.js

```diff
@@ -0,0 +1,97 @@
+import {
+  editDashboard,
+  modal,
+  restore,
+  visitDashboard,
+  visitQuestion,
+} from "e2e/support/helpers";
+
+const baseQuestion = {
+  name: "Base question",
+  display: "scalar",
+  native: {
+    query: "SELECT 1",
+  },
+};
+
+const invalidQuestion = {
+  name: "Invalid question",
+  display: "scalar",
+  native: {
+    query: "SELECT --1",
```

Comment:
This looks different from the step provided for 45.x where we create a valid question first, added it to another card then make it invalid.

```ts
  native: { query: "select 2" },
```

I believe this should be
```suggestion
    query: "SELECT 1",
```

Reply:
Fixed in https://github.com/metabase/metabase/pull/33328/commits/196d72150fb1c855caaf795061d875df48ac2283